src/iidm/ValidationUtils.cpp

```diff
@@ -385,7 +385,10 @@ long checkTapPosition(const Validable& validable, long tapPosition, long lowTapP
     return tapPosition;
 }
 
-double checkTargetDeadband(const Validable& validable, double targetDeadband) {
+double checkTargetDeadband(const Validable& validable, const std::string& validableType, bool regulating, double targetDeadband) {
+    if (regulating && std::isnan(targetDeadband)) {
+        throw ValidationException(validable, "Undefined value for target deadband of regulating " + validableType);
+    }
     if (!std::isnan(targetDeadband) && targetDeadband < 0) {
```

Comment:
I think the first part of this test is redundant: `NaN < 0` should return false.